konfera/register/views.py

```diff
@@ -9,13 +9,13 @@
 from konfera.register.forms import RegistrationForm
 
 
-def private_registration(request, event_slug, ticket_uuid):
+def private_registration(request, slug, ticket_uuid):
```

Comment:
private_registration and public_registration share a lot of same code base, how about we do one generic function for the stuff that is shared. We will have multiple view that will call this generic function with different parameters.